bundles/plugins/org.bonitasoft.studio.common/src/org/bonitasoft/studio/common/properties/AbstractBonitaDescriptionSection.java

```diff
@@ -14,6 +14,8 @@
  */
 package org.bonitasoft.studio.common.properties;
 
+import static org.eclipse.jface.layout.GridDataFactory.fillDefaults;
```

Comment:
I agree on this specific one (GridDataFactory and GridLayoutFactory are both using create() so it can be confusing)
However, when using guava, it seems to quite common and to me, not harmful for the readability.